404.html

```diff
@@ -0,0 +1,152 @@
+---
+layout: default
+title: Page Not Found
+canonical_url: /404.html
+---
+{% include nav.html %}
+<main class="front-section">
+  <div class="container">
+    <div class="error-content">
+      <div class="error-code">404</div>
+      <h1>Page Not Found</h1>
```

Comment:
why is `error-content` needed? why can't we use the standard section class names. also not clear why 404 is in its own div
```suggestion
    <div class="...">
      <h1>404 Page Not Found</h1>
```

Reply:
ah yes, you're right. switched to using .masthead instead and put the 404 in .lead.